### ISSUE TITLE
RGB theme is no longer RGB

### ISSUE THREAD

**Issue by devkennyy** [devkennyy/rungeon#147]
The RGB theme doesn't actually make any of the stage content RGB. The selectors are there but there's obviously a problem. 
```css
#index.theme-rgb .nav-link,
#index.theme-rgb .stage .icon,
#index.theme-rgb .doors i,
#index.theme-rgb .links i,
#index.theme-rgb #themePopup,
#index.theme-rgb #themePopup #closeButton {
  animation: rgb 5s linear infinite !important;
}
```

**Comment by Shabbyconnor** [comment]
It appears that the CSS applied to the "stage" class is overwriting the RGB. Turn it off and we get the RGB. However, the CSS is reasonable for applying the rest of the themes so I'll have to figure something out

**Comment by Shabbyconnor** [comment]
Kay. I've got a fix. Remind me what RGB is supposed to look like. Does the text need to be RGB too? How about the continue button?

**Comment by devkennyy** [comment]
There's an RGB animation, you just need to apply it to the stage icon, whatever looks good really.

**Comment by Shabbyconnor** [comment]
Originally it looked like this:
<img width="1440" alt="Screen Shot 2022-07-28 at 4 10 33 PM" src="https://user-images.githubusercontent.com/109128466/181646535-2b40d60f-388d-420b-ac0f-f2d9cbedc746.png">
I can make it look like that, or I can go more intense and make it look like this:
<img width="1440" alt="Screen Shot 2022-07-28 at 4 17 36 PM" src="https://user-images.githubusercontent.com/109128466/181646682-510cf44b-3715-446f-b9fc-159d33fa4d01.png">
What do you think?

**Comment by devkennyy** [comment]
This is definitely nicer, if you want rgb, you're getting rgb :trollface:

**Comment by GruelingPine185** [comment]
@Shabbyconnor Was it intentional to have the elements and icons changing colors unsynchronized?

**Comment by Shabbyconnor** [comment]
No, but I'm not sure of a way to sync them up. 
The problem is the animation starts at the beginning when a new element is created. Some elements on the page never change like the navbar. But some like the text and icon change every time the stage switches.

**Comment by GruelingPine185** [comment]
Oh. So since only the icon and the prompt changes every stage switch, could it be possible to make a function that captures stage switches and removes and replaces the rgb animation? The transition between stages might be a bit rough, but at least the colors are synced

**Comment by Shabbyconnor** [comment]
Yeah. You'd have to replace the animation for each element though.

**Comment by Shabbyconnor** [comment]
> @Shabbyconnor Was it intentional to have the elements and icons changing colors unsynchronized?

Fixed by #151
